apps/authentication-service/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@repo/shared-utils": "*",
+    "@repo/backend-utils": "*",
```

Comment:
I added this new repo package for backend specific utils because when I tried adding some things which are used across all the backend services, the front-end code complained.

For example my backend function requires this package `express-oauth2-jwt-bearer`, but if you try to install that package on the front-end code then it says you need a bunch of polyfills.